server/etcdserver/api/v3compactor/periodic_test.go

```diff
@@ -172,3 +173,99 @@ func TestPeriodicPause(t *testing.T) {
 		t.Errorf("compact request = %v, want %v", a[0].Params[0], wreq.Revision)
 	}
 }
+
+func TestPeriodicSkipRevNotChangeHourly(t *testing.T) {
```

Comment:
Why do we add a test for both hours and minutes? Is there a expected difference? From comments compaction didn't happens.